index.html

```diff
@@ -35,12 +35,17 @@
                 height: 100%;
                 width: 90%;
                 font-size: 20px;
+                display: none;
```

Comment:
This looks great!

I wanted to get your input. Should we have the focus cycle through all the elements?

For example, if you're on the last element and press tab, should it stay stuck on the last element? Or should it go to the first element?

![modal trap](https://user-images.githubusercontent.com/14035529/48653453-4b918780-e9d3-11e8-9ab7-9366b2806545.gif)

The GIF is from the example in [wai-aria-practices](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html).

Reply:
As a user, I would expect it to go to the first element.